lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -9,14 +9,20 @@ import {
     JoseHeader,
     Logger,
     PerformanceEvents,
-    PkceCodes,
     SignedHttpRequest,
     SignedHttpRequestParameters,
 } from "@azure/msal-common";
 import { base64Encode, urlEncode, urlEncodeArr } from "../encode/Base64Encode";
 import { base64Decode } from "../encode/Base64Decode";
-import { PkceGenerator } from "./PkceGenerator";
-import { BrowserCrypto } from "./BrowserCrypto";
+import {
+    createNewGuid,
+    exportJwk,
+    generateKeyPair,
+    importJwk,
+    sha256Digest,
+    sign,
+    validateCryptoAvailable,
+} from "./BrowserCrypto";
```

Comment:
Can we do namespaces or modules and invoke any function in the namespace/module instead of importing each function? I think as we do more of this modeling for all utils, the imports will get too long.

Reply:
Does adding namespaces negatively impacts tree shaking?

Reply:
It does not. It's a good suggestion, I'll make the change here. My general opinion is if we're importing just one or two things I think it's fine to do so, anything more should be namespaced.

Reply:
Done, thanks for the suggestion!